### PR TITLE
Guid readable

### PR DESCRIPTION
**Pull request by GalRabinDemisto**
## Status
Ready

## Related Issues
https://github.com/demisto/etc/issues/19289

## Description
Change id to readable id by following constraints:
1. only js scripts.
2. only not deprecated.
3. id isn't readable.

Change Regex when validating files:
1. regex for scripts.
2. regex for integrations.

Change update playbook:
1. When updating playbook, code will be as a code block.
2. Support python3, until now was python2.

## Does it break backward compatibility?
   - No

I used the following script:
```python3
import os
import glob
import yaml, yamlordereddictloader
import re

def add_change_log(file_path):
    ch_file_name = file_path.split('.')[0] + '_CHANGELOG.md'
    with open(ch_file_name, mode='w') as f:
        f.write('-')

def change_file(file_path):
    data = {}
    with open(file_path, mode='r') as f:
        data = yaml.load(f, Loader=yamlordereddictloader.SafeLoader)
        data['commonfields']['id'] = data['name']
        data['commonfields']['version'] = -1

    # Configure safe dumper (multiline for strings)
    yaml.SafeDumper.org_represent_str = yaml.SafeDumper.represent_str
    def repr_str(dumper, data):
        if '\n' in data:
            return dumper.represent_scalar(u'tag:yaml.org,2002:str', data, style='|')
        return dumper.org_represent_str(data)
    yaml.add_representer(str, repr_str, Dumper=yamlordereddictloader.SafeDumper)


    with open(file_path, 'w') as f:
        yaml.dump(
            data,
            f,
            Dumper=yamlordereddictloader.SafeDumper,
            default_flow_style=False)

def validate_req(file_path):
    with open(file_path, mode='r') as f:
        valid = True
        data = yaml.load(f, Loader=yaml.Loader)
        # JavaScript validation
        if data['type'] != 'javascript':
            return False
        # ID wrong structure
        wrong_id = re.compile(r'.*-.*-.*-.*-.*')
        if not wrong_id.search(data['commonfields']['id']):
            return False
        # Deprecated
        if data.get('deprecated'):
            return False
    return True

def main():
    os.chdir(r'/Users/grabin/dev/demisto/content')
    for file_path in glob.iglob('/Users/grabin/dev/demisto/content/Scripts/*.yml'):
        if validate_req(file_path):
            change_file(file_path)
            add_change_log(file_path)
            print(file_path)

if __name__ == '__main__':
    main()
```

### CODE REVIEW

**Comment by GalRabinDemisto** [comment]
@yaakovi 
Hi,
As requested, I validated that the slash isn't showing on the UI .

**Comment by GalRabinDemisto** [comment]
@kirbles19 
Hi,
I changed the ID of 5 scripts, Inorder to that I added an empty changelog that you need to approve, I will appreciate if you do that.

Thanks

File: Tests/scripts/constants.py [hunk 1]
```diff
@@ -61,7 +61,7 @@
 CHECKED_TYPES_REGEXES = [PLAYBOOK_REGEX, INTEGRATION_YML_REGEX, BETA_INTEGRATION_YML_REGEX,
                          WIDGETS_REGEX, DASHBOARD_REGEX, CONNECTIONS_REGEX, CLASSIFIER_REGEX, SCRIPT_YML_REGEX,
                          LAYOUT_REGEX, INCIDENT_FIELD_REGEX, MISC_REGEX, REPORT_REGEX,
-                         REPUTATION_REGEX]
+                         REPUTATION_REGEX, INTEGRATION_REGEX, SCRIPT_REGEX]
```
Comment (yaakovi) [inline Tests/scripts/constants.py]: this change is already in master, pull and fix the conflicts

**Comment by GalRabinDemisto** [comment]
@yaakovi 
Hi,
Change reame file by convention done.

**Pull request merged**